services/uncoolifyService.js

```diff
@@ -367,6 +367,35 @@ uncoolifyDicts.unicodemix = {
 	'ℨ':'z',
 }
 
+uncoolifyDicts.newemoji={
+	'🥨'='a',
```

Comment:
That's  not a dictionary, it should be
```
{
    'key': 'value'
}
```

Reply:
> That's not a dictionary, it should be
> 
> ```
> {
>     'key': 'value'
> }
> ```

what does that mean?

Reply:
Please run the app and test all corner case each time you commit.

Reply:
> Please run the app and test all corner case each time you commit.

this time its all set.
please have a look!

Reply:
the curly bracket should end like this?
{
 'key':'value'
};

Reply:
Some of the dictionary end with 

} ;
and some do not?

Reply:
Everything is checked and done properly.
Please review once!